short_answer/static/js/src/short_answer.js

```diff
@@ -185,8 +206,26 @@ function ShortAnswerXBlock(runtime, element, options) {
     });
   });
 
+  /**
+   * Enable/Disable the submit button.
+   */
+  $shortAnswer.keyup(function() {
+    if (isSubmitAllowed()) {
+      $answerSubmitButton.removeAttr('disabled');
+    }
+    else {
+      $answerSubmitButton.attr('disabled', true);
+    }
+  });
+
   $(function($) {
     const csvDownloadUrl = runtime.handlerUrl(element, 'csv_download');
     $('form[name=csv-download]', element).attr('action', csvDownloadUrl);
+
+    $shortAnswer.keyup();
+
+    if (shouldDisableAnswerTextArea) {
+      $shortAnswer.attr('disabled', true);
+    }
```

Comment:
nitpick: I think you can omit the `{}`

also: $shortAnswer.attr('disabled', shouldDisableAnswerTextArea);